src/status_im/chat/views/api/choose_contact.cljs

```diff
@@ -15,17 +15,14 @@
 (defn- select-contact [arg-index bot-db-key {:keys [name] :as contact}]
   (let [contact (select-keys contact [:address :whisper-identity :name :photo-path :dapp?])
         name    (str/replace name (re-pattern const/arg-wrapping-char) "")]
-    (dispatch [:set-command-argument [arg-index name true]])
-    (dispatch [:set-in-bot-db {:path  [:public (keyword bot-db-key)]
-                               :value contact}])
-    (dispatch [:select-next-argument])))
+    (dispatch [:set-contact-as-command-argument bot-db-key contact arg-index name])))
```

Comment:
Yes, that's exactly what does it mean. Unfortunately in some cases, we still can't straight away remove those  partial events because another places in app still use them, but we will eventually come to that (eq there is no fundamental reason for those partial db operations to exist as distinct events).